src/algorithms/PVT/gnuradio_blocks/rtklib_pvt_gs.cc

```diff
@@ -174,7 +181,7 @@ rtklib_pvt_gs::rtklib_pvt_gs(uint32_t nchannels,
 
     d_type_of_rx = conf_.type_of_receiver;
 
-    // GPS Ephemeris data message port in
+    // GPS Ephemeris data message pspoofing_detectorort in
```

Comment:
I hope this is just keyboard dribble, and not an intentional edit to the comment?